packages/conform-dom/formdata.ts

```diff
@@ -64,7 +64,7 @@ export function getPaths(name: string | undefined): Array<string | number> {
  * const name = formatPaths(['todos', 0, 'content']); // "todos[0].content"
  * ```
  */
-export function formatPaths(paths: Array<string | number>): string {
+export function formatPaths(paths: Array<PropertyKey>): string {
```

Comment:
I am not sure if we could support `symbol` here. We might be able to serialize it in some form but we will not be able restore it back as a symbol. 

We might have to throw an error with zod 4 if the paths contain any symbol.

Reply:
fixed